x-pack/plugin/esql/qa/testFixtures/src/main/resources/k8s-timeseries.csv-spec

```diff
@@ -201,6 +201,24 @@ max(rate(network.total_bytes_in)):double | time_bucket:datetime     | cluster:ke
 11.562737642585551                       | 2024-05-10T00:10:00.000Z | prod
 ;
 
+maxRateWithInlineFilter
+required_capability: metrics_command
+TS k8s | STATS max_rate = max(rate(network.total_bytes_in)) WHERE cluster=="prod" BY cluster | SORT cluster;
+
+max_rate:double         | cluster:keyword
+8.716707021791768       | prod
+null                    | qa
+null                    | staging
```

Comment:
We cannot rewrite a single stats operation with an inline filter to a pre-filter, because aggregation with an inline filter may produce groups for which the data does not match the filter. Should we break this behavior in time-series aggregations?